src/handler/http/request/dashboards/mod.rs

```diff
@@ -308,45 +282,3 @@ fn get_folder(req: HttpRequest) -> String {
     let query = web::Query::<HashMap<String, String>>::from_query(req.query_string()).unwrap();
     crate::common::utils::http::get_folder(&query)
 }
-
-/// Parses the bytes into a dashboard with the given version.
-fn parse_dashboard(bytes: web::Bytes) -> Result<Dashboard, serde_json::Error> {
-    let d_version: DashboardVersion = json::from_slice(&bytes)?;
-    let version = d_version.version;
```

Comment:
This logic now happens inside of the various implementations of the `TryFrom` trait that are implemented in `http/models/dashboards` to transform "HTTP Layer" models into "Service Layer" models